docs/software.md

```diff
@@ -0,0 +1,61 @@
+# Software
+
+This page lists general purpose open source software that manipulates HGVS
+expressions. Our intention is to help the community find software that aids in
+the adoption of HGVS Nomenclature. The entries below are provided by the
+authors; the entries are not validated or endorsed by the HVNC. Software packages are listed
+alphabetically.  See [Listing Definitions](#listing-definitions) for an
+explanation of terms. See the [submission requirements and
+instructions](#submissions) for information about submitting to this list.
+
+
+```sh exec="true"
+bin/make-software-table docs/software/*.yml
+```
+
+## Listing Definitions
+
+- **Software Type**: One of:
+    - Library: Software component that is used by developers when building applications
+    - API (e.g., REST): Software that exposes an internet-visible programming interface that is language-independent
+    - Web UI: Software used with a browser window and generally intended for broader audiences
+
+- **License**: Only open-source software with licenses approved by the [Open Source
+  Initiative](https://opensource.org) will be listed. In addition,
+  [well-recognized
+  licenses](https://opensource.org/licenses/?categories=popular-strong-community)
+  will be preferred.
+
+- **Functionality**: Software is annotated as performing one or more of the following functions:
+    - conversion: Does the software convert to/from HGVS and other formats (e.g., VCF, SPDI, VRS)
+    - liftover: Does the software provide *direct* liftover between aligned genomic sequences?
+    - normalization: Does the software normalize variants (e.g., 3' shift) and rewrite (e.g., `ins` → `dup`)
+    - parsing: Does the software parse HGVS expressions into a usable structure for analysis?
+    - projection: Does the software project variants from one sequence to another?
+    - translation: Does the software infer protein consequences from transcript variants?
+    - validation: Does the software provide syntactic validation (i.e., the correct form) and semantic validation (i.e., the parsed structure makes sense)?
+
```

Comment:
We should provide criteria around what belongs in the `description` section, if we are to have one. This will help keep software descriptions balanced.

Reply:
Okay, maybe that's the easiest thing to do right now. I'm looking for easy wins that don't create lots of debate and deliberation. So, I'll remove the description column.